### DISCUSSION TITLE
Monitoring

### DISCUSSION THREAD

**Discussion by ptacquet** [DaxStudio/DaxStudio#766] [General]
Hi guys,

Ca we analyse a dataset refresh with dax studio ?
I tried with "all queries" but nothing appears.

More generally, how can we monitor the dataset refresh process ?

Thks.

**Comment by ptacquet** [comment]
Important precision: we use ppu licences,  premium metrics apps aren't available.

**Reply by dgosbell** [reply]
No, you cannot do this at the moment.

You can capture information about data set refreshes using a technique like the one outlined here https://dax.tips/2021/02/15/visualise-your-power-bi-refresh/ I've started looking at incorporating something like this into DAX Studio, but it's still a work in progress at the moment.
